package.json

```diff
@@ -43,7 +43,10 @@
     "cypress": "3.2.0",
     "eslint": "^6.3.0",
     "eslint-config-prettier": "^6.2.0",
+    "eslint-config-react-app": "^5.1.0",
+    "eslint-plugin-flowtype": "^4.5.2",
     "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-prettier": "^3.1.0",
```

Comment:
can we remove any of these explicit dependencies now? let's make sure anything removed from eslintrc are also removed from our dev dependencies

Reply:
I double checked over our `plugins`, and it looks like everything we explicitly removed from `.eslintrc` was also required as a peer dependency to get `eslint-config-react-app` to start